po/fr.po

```diff
@@ -13021,208 +13092,297 @@ msgstr[1] ""
+msgid "(private)"
+msgstr "privé"
+
+#: ../src/libs/tagging.c:2105
 msgid "Select a keyword file"
 msgstr "sélectionner un fichier de mot-clé"
 
-#: ../src/libs/tagging.c:1247
+#: ../src/libs/tagging.c:2108
 msgid "_import"
 msgstr "_importer"
 
-#: ../src/libs/tagging.c:1261
+#: ../src/libs/tagging.c:2122
 msgid "error importing tags"
 msgstr "erreur lors de l'import de mot-clé"
 
-#: ../src/libs/tagging.c:1263
+#: ../src/libs/tagging.c:2124
 #, c-format
 msgid "%zd tags imported"
 msgstr "« %zd » mots-clés importés"
 
-#: ../src/libs/tagging.c:1284
+#: ../src/libs/tagging.c:2146
 msgid "Select file to export to"
 msgstr "sélectionne le fichier pour l'export"
 
-#: ../src/libs/tagging.c:1287
+#: ../src/libs/tagging.c:2149
 msgid "_export"
 msgstr "_exporter"
 
-#: ../src/libs/tagging.c:1302
+#: ../src/libs/tagging.c:2164
 msgid "error exporting tags"
 msgstr "erreur lors de l'export de mot-clé"
 
-#: ../src/libs/tagging.c:1304
+#: ../src/libs/tagging.c:2166
 #, c-format
 msgid "%zd tags exported"
 msgstr "« %zd » mots-clés exportés"
 
-#: ../src/libs/tagging.c:1453
+#: ../src/libs/tagging.c:2524
 msgid ""
 "attached tags,\n"
-"double-click to detach"
+"double-click to detach\n"
+"right-click for other actions on attached tag,\n"
+"Ctrl-wheel scroll to resize the window"
 msgstr ""
-"mots-clés attachés,\n"
-"double-clic pour détacher"
+"mots-clés liés,\n"
+"double-clic pour attacher\n"
+"clic-droit pour afficher le menu des actions du mot-clé sélectionné\n"
+"Ctrl+molette souris pour redimensionner la fenêtre"
 
-#: ../src/libs/tagging.c:1461
+#: ../src/libs/tagging.c:2535
 msgid "attach"
 msgstr "attacher"
 
-#: ../src/libs/tagging.c:1463
+#: ../src/libs/tagging.c:2538
 msgid "attach tag to all selected images"
 msgstr ""
 "attache le mot-clé\n"
 "à toutes les images sélectionnées"
 
-#: ../src/libs/tagging.c:1468
+#: ../src/libs/tagging.c:2543
 msgid "detach"
 msgstr "détacher"
 
-#: ../src/libs/tagging.c:1470
+#: ../src/libs/tagging.c:2546
 msgid "detach tag from all selected images"
 msgstr ""
 "détache le mot-clé\n"
 "de toutes les images sélectionnées"
 
-#: ../src/libs/tagging.c:1483
+#: ../src/libs/tagging.c:2553
+msgid "toggle list with / without hierarchy"
+msgstr "active/désactive la liste hiérarchique"
+
+#: ../src/libs/tagging.c:2561
+msgid "toggle sort by name or by count"
+msgstr "activer le classement par nom ou par nombre"
+
+#: ../src/libs/tagging.c:2571
+msgid "toggle show or not darktable tags"
+msgstr "activer/désactiver l'affichage des mots-clés spécifiques à darktable"
+
+#: ../src/libs/tagging.c:2587
 msgid "enter tag name"
 msgstr "entrez un mot-clé"
 
-#: ../src/libs/tagging.c:1525
+#: ../src/libs/tagging.c:2597
+msgid "clear entry"
+msgstr "effacer texte"
+
+#: ../src/libs/tagging.c:2649
 msgid ""
-"related tags,\n"
+"tag dictionary,\n"
 "double-click to attach,\n"
-"right-click for other actions on selected tag"
+"right-click for other actions on selected tag,\n"
+"Ctrl-wheel scroll to resize the window"
```

Comment:
No C capital letter here.

Reply:
I've only respected how it was written in english. If you see the source, in english it is also written with a capital letter. I change that on my translation. For english one, I let you change that.